### PR TITLE
Annual report

### PR DESCRIPTION
**Pull request by seotts**
### Resolves:

_What Github issue does this resolve (please include link)? Please do not submit PRs that only partially implement an issue. Please do not submit PRs that are not associated with a Github issue, or that only partially implement an issue._

### Changes:

_Describe what this Pull Request does. Be sure to include a brief description of the issue the Pull Requests solves too._

### Test Coverage:

_Please show how you have added tests to cover your changes or describe how you have tested the changes (include a screenshot if possible)._

### CODE REVIEW

File: src/routes.json [hunk 1]
```diff
@@ -6,6 +6,14 @@
         "view": "about/about",
         "title": "About"
     },
+    {
+        "name": "annual-report",
+        "pattern": "^/annual-report/?$",
+        "routeAlias": "/annual-report/?$",
+        "view": "annual-report/annual-report",
+        "title": "annual-report",
+        "viewportWidth": "device-width"
```
Comment (seotts) [inline src/routes.json:15]: I'm not sure if we need this or exactly what it does. We added it when we were trying to fix issues we were having with responsiveness.

File: src/views/annual-report/annual-report.jsx [hunk 2]
```diff
@@ -0,0 +1,1279 @@
...
+
+        return {
+            height,
+            offsetTop,
+            offsetBottom
+        };
+    }
+
+    // While scrolling, update the subnav to reflect the currently visible section
+    handleScroll () {
```
Comment (seotts) [inline src/views/annual-report/annual-report.jsx:136]: Everything involving the scrolling and the subnav bar are probably the most complex part of the code, especially this function.

File: src/views/annual-report/annual-report.jsx [hunk 3]
```diff
@@ -0,0 +1,1279 @@
...
+                    )}
+                    onClick={this.handleSubnavItemClick(SECTIONS.donate)}
+                >
+                    <FormattedMessage id="annualReport.subnavDonate" />
+                </a>
+            </FlexRow>);
+
+        return (
+            <div>
+                <div
```
Comment (seotts) [inline src/views/annual-report/annual-report.jsx:242]: We should probably pull the subnav into a separate component, after the launch of the annual report.

Either way, the way we are handling responsiveness here might be a little confusing. For small screens, we render both the section indicator (in the "top bar") and the drop down of list of sections (in the "bottom bar.") For small screens, we render only the list of sections, in the top bar (with different styling)

File: src/views/annual-report/annual-report.jsx [hunk 4]
```diff
@@ -0,0 +1,1279 @@
...
+                                        </div>
+                                    </div>
+                                </div>
+                                <img
+                                    className="graph"
+                                    src="/images/annual-report/expenses-graph.svg"
+                                />
+                            </div>
+                            <div className="financials-button-wrapper">
+                                <a href="https://secure.donationpay.org/scratchfoundation/">
```
Comment (seotts) [inline src/views/annual-report/annual-report.jsx:865]: we need to replace this with a link to the financials pdf

File: src/views/annual-report/annual-report.jsx [hunk 5]
```diff
@@ -0,0 +1,1279 @@
...
+                                            <li>Warner Media</li>
+                                        </ul>
+                                    </div>
+                                </div>
+                                <div className="supporters-level">
+                                    <h5>
+                                        <FormattedHTMLMessage id="annualReport.supportersCollaborationTitle" />
+                                    </h5>
+                                    <hr />
+                                    <div className="supporters-list">
```
Comment (seotts) [inline src/views/annual-report/annual-report.jsx:973]: I wonder if these long lists of donors would be better in a separate JSON? Maybe in a follow-up pr after release

File: src/views/annual-report/country-usage.json [hunk 6]
```diff
@@ -0,0 +1 @@
...":12.1142533654},"Uruguay":{"code":"uy","display":"Uruguay","count":61137,"log count":11.0208725265},"Uzbekistan":{"code":"uz","display":"Uzbekistan","count":6205,"log count":8.7331106976},"Vanuatu":{"code":"vu","display":"Vanuatu","count":2290,"log count":7.7363070965},"Vatican City":{"code":"va","display":"Vatican City","count":2767,"log count":7.9255189798},"Venezuela":{"code":"ve","display":"Venezuela","count":27878,"log count":10.2355931263},"Vietnam":{"code":"vn","display":"Vietnam","count":105419,"log count":11.5656981645},"Virgin Islands, British":{"code":"vg","display":"Virgin Islands, British","count":5573,"log count":8.6256887876},"Virgin Islands, U.S.":{"code":"vi","display":"Virgin Islands, U.S.","count":11751,"log count":9.3716936223},"Wallis and Futuna":{"code":"wf","display":"Wallis and Futuna","count":2209,"log count":7.7002952034},"Western Sahara":{"code":"eh","display":"Western Sahara","count":2255,"log count":7.7209052519},"Yemen":{"code":"ye","display":"Yemen","count":8123,"log count":9.0024548231},"Zambia":{"code":"zm","display":"Zambia","count":8026,"log count":8.9904415508},"Zimbabwe":{"code":"zw","display":"Zimbabwe","count":39471,"log count":10.583321504}}
```
Comment (seotts) [inline src/views/annual-report/country-usage.json:1]:
```suggestion
{"Afghanistan":{"code":"af","display":"Afghanistan","count":235212,"log count":12.3682425141},"Aland Islands":{"code":"ax","display":"\u00c5land Islands","count":180538,"log count":12.1036965609},"Albania":{"code":"al","display":"Albania","count":419606,"log count":12.9470714547},"Algeria":{"code":"dz","display":"Algeria","count":193712,"log count":12.1741277989},"American Samoa":{"code":"as","display":"American Samoa","count":64164,"log count":11.0691975847},"Andorra":{"code":"ad","display":"Andorra","count":71292,"log count":11.1745393981},"Angola":{"code":"ao","display":"Angola","count":39310,"log count":10.5792342184},"Anguilla":{"code":"ai","display":"Anguilla","count":32696,"log count":10.3950080252},"Antigua and Barbuda":{"code":"ag","display":"Antigua and Barbuda","count":23263,"log count":10.0546193945},"Argentina":{"code":"ar","display":"Argentina","count":200620,"log count":12.2091678504},"Armenia":{"code":"am","display":"Armenia","count":35896,"log count":10.4883811477},"Aruba":{"code":"aw","display":"Aruba","count":25047,"log count":10.1285093389},"Australia":{"code":"au","display":"Australia","count":1655244,"log count":14.319458988},"Austria":{"code":"at","display":"Austria","count":93575,"log count":11.4465185328},"Azerbaijan":{"code":"az","display":"Azerbaijan","count":28601,"log count":10.2611969612},"Bahamas":{"code":"bs","display":"Bahamas","count":35542,"log count":10.4784703747},"Bahrain":{"code":"bh","display":"Bahrain","count":13287,"log count":9.4945413926},"Bangladesh":{"code":"bd","display":"Bangladesh","count":27348,"log count":10.216398679},"Barbados":{"code":"bb","display":"Barbados","count":7959,"log count":8.9820586428},"Belarus":{"code":"by","display":"Belarus","count":39263,"log count":10.5780378786},"Belgium":{"code":"be","display":"Belgium","count":210229,"log count":12.2559526918},"Belize":{"code":"bz","display":"Belize","count":7418,"log count":8.911664758},"Benin":{"code":"bj","display":"Benin","count":3697,"log count":8.2152769589},"Bermuda":{"code":"bm","display":"Bermuda","count":9432,"log count":9.1518634422},"Bhutan":{"code":"bt","display":"Bhutan","count":4100,"log count":8.3187422527},"Bolivia":{"code":"bo","display":"Bolivia","count":21267,"log count":9.9649118545},"Bonaire, Sint Eustatius and Saba":{"code":"bq","display":"Bonaire, Sint Eustatius and Saba","count":193943,"log count":12.1753195804},"Bosnia and Herzegovina":{"code":"ba","display":"Bosnia and Herzegovina","count":13386,"log count":9.5019646637},"Botswana":{"code":"bw","display":"Botswana","count":4340,"log count":8.3756296271},"Bouvet Island":{"code":"bv","display":"Bouvet Island","count":2457,"log count":7.8066963725},"Brazil":{"code":"br","display":"Brazil","count":597136,"log count":13.2999001721},"British Indian Ocean Territory":{"code":"io","display":"British Indian Ocean Territory","count":8812,"log count":9.0838697079},"Brunei":{"code":"bn","display":"Brunei Darussalami","count":6161,"log count":8.725994381},"Bulgaria":{"code":"bg","display":"Bulgaria","count":58926,"log count":10.9840376984},"Burkina Faso":{"code":"bf","display":"Burkina Faso","count":2768,"log count":7.9258803167},"Burundi":{"code":"bi","display":"Burundi","count":2017,"log count":7.609366538},"Cambodia":{"code":"kh","display":"Cambodia","count":11489,"log count":9.3491453349},"Cameroon":{"code":"cm","display":"Cameroon","count":5445,"log count":8.6024530354},"Canada":{"code":"ca","display":"Canada","count":1286373,"log count":14.0673371884},"Cape Verde":{"code":"cv","display":"Cape Verde","count":2997,"log count":8.0053670673},"Cayman Islands":{"code":"ky","display":"Cayman Islands","count":3257,"log count":8.0885618053},"Central African Republic":{"code":"cf","display":"Central African Republic","count":1892,"log count":7.5453897496},"Chad":{"code":"td","display":"Chad","count":3963,"log count":8.2847565932},"Chile":{"code":"cl","display":"Chile","count":120888,"log count":11.7026197761},"China":{"code":"cn","display":"China","count":3001455,"log count":14.9146077291},"Christmas Island":{"code":"cx","display":"Christmas Island","count":14149,"log count":9.5573992292},"Cocos (Keeling) Islands":{"code":"cc","display":"Cocos (Keeling) Islands","count":3057,"log count":8.0251893219},"Colombia":{"code":"co","display":"Colombia","count":480593,"log count":13.0827760371},"Comoros":{"code":"km","display":"Comoros","count":1649,"log count":7.4079243226},"Congo":{"code":"cg","display":"Congo","count":6199,"log count":8.7321432677},"Congo, The Democratic Republic of the":{"code":"cd","display":"The Democratic Republic of the Congo","count":3929,"log count":8.2761402196},"Cook Islands":{"code":"ck","display":"Cook Islands","count":3347,"log count":8.1158197012},"Costa Rica":{"code":"cr","display":"Costa Rica","count":29078,"log count":10.2777371535},"Cote d'Ivoire":{"code":"ci","display":"C\u00f4te d'Ivoire","count":7428,"log count":8.9130119225},"Croatia":{"code":"hr","display":"Croatia","count":26379,"log count":10.180323518},"Cuba":{"code":"cu","display":"Cuba","count":9570,"log count":9.1663884844},"Curacao":{"code":"cw","display":"Cura\u00e7ao","count":268,"log count":5.5909869805},"Cyprus":{"code":"cy","display":"Cyprus","count":9822,"log count":9.1923800466},"Czech Republic":{"code":"cz","display":"Czech Republic","count":78986,"log count":11.2770259006},"Denmark":{"code":"dk","display":"Denmark","count":206640,"log count":12.2387334278},"Djibouti":{"code":"dj","display":"Djibouti","count":4719,"log count":8.4593521917},"Dominica":{"code":"dm","display":"Dominica","count":3272,"log count":8.0931566977},"Dominican Republic":{"code":"do","display":"Dominican Republic","count":23553,"log count":10.0670084799},"Ecuador":{"code":"ec","display":"Ecuador","count":90792,"log count":11.416326455},"Egypt":{"code":"eg","display":"Egypt","count":65590,"log count":11.0911785243},"El Salvador":{"code":"sv","display":"El Salvador","count":25798,"log count":10.1580522485},"Equatorial Guinea":{"code":"gq","display":"Equatorial Guinea","count":1608,"log count":7.3827464497},"Eritrea":{"code":"er","display":"Eritrea","count":3484,"log count":8.155936338},"Estonia":{"code":"ee","display":"Estonia","count":43970,"log count":10.6912628622},"Swaziland":{"code":"sz","display":"Eswatini","count":3266,"log count":8.0913212735},"Ethiopia":{"code":"et","display":"Ethiopia","count":7630,"log count":8.9398431243},"Falkland Islands (Malvinas)":{"code":"fk","display":"Falkland Islands (Malvinas)","count":1550,"log count":7.3460102099},"Faroe Islands":{"code":"fo","display":"Faroe Islands","count":1508,"log count":7.3185395486},"Fiji":{"code":"fj","display":"Fiji","count":11010,"log count":9.3065592297},"Finland":{"code":"fi","display":"Finland","count":133785,"log count":11.8039893128},"France":{"code":"fr","display":"France","count":906888,"log count":13.7177742375},"French Guiana":{"code":"gf","display":"French Guiana","count":2915,"log count":7.9776250988},"French Polynesia":{"code":"pf","display":"French Polynesia","count":2029,"log count":7.6152983398},"French Southern Territories":{"code":"tf","display":"French Southern Territories","count":1454,"log count":7.2820736581},"Gabon":{"code":"ga","display":"Gabon","count":3190,"log count":8.0677761958},"Gambia":{"code":"gm","display":"Gambia","count":2327,"log count":7.7523351633},"Georgia":{"code":"ge","display":"Georgia","count":35875,"log count":10.4877959531},"Germany":{"code":"de","display":"Germany","count":277400,"log count":12.5332157869},"Ghana":{"code":"gh","display":"Ghana","count":9826,"log count":9.1927872127},"Gibraltar":{"code":"gi","display":"Gibraltar","count":2636,"log count":7.8770178956},"Greece":{"code":"gr","display":"Greece","count":112750,"log count":11.6329282574},"Greenland":{"code":"gl","display":"Greenland","count":2660,"log count":7.8860814018},"Grenada":{"code":"gd","display":"Grenada","count":1980,"log count":7.5908521237},"Guadeloupe":{"code":"gp","display":"Guadeloupe","count":3456,"log count":8.1478671299},"Guam":{"code":"gu","display":"Guam","count":3200,"log count":8.0709060888},"Guatemala":{"code":"gt","display":"Guatemala","count":28611,"log count":10.2615465382},"Guernsey":{"code":"gg","display":"Guernsey","count":6352,"log count":8.7565250029},"Guinea":{"code":"gn","display":"Guinea","count":1556,"log count":7.3498737047},"Guinea-Bissau":{"code":"gw","display":"Guinea-Bissau","count":946,"log count":6.8522425691},"Guyana":{"code":"gy","display":"Guyana","count":2267,"log count":7.7262126505},"Haiti":{"code":"ht","display":"Haiti","count":4645,"log count":8.4435466512},"Heard Island and McDonald Islands":{"code":"hm","display":"Heard Island and McDonald Islands","count":9055,"log count":9.1110723703},"Honduras":{"code":"hn","display":"Honduras","count":10614,"log count":9.2699291634},"Hong Kong":{"code":"hk","display":"Hong Kong","count":219008,"log count":12.2968635378},"Hungary":{"code":"hu","display":"Hungary","count":97047,"log count":11.4829506762},"Iceland":{"code":"is","display":"Iceland","count":17679,"log count":9.7801327735},"India":{"code":"in","display":"India","count":316221,"log count":12.6641966152},"Indonesia":{"code":"id","display":"Indonesia","count":67702,"log count":11.1228710006},"Iran":{"code":"ir","display":"Iran","count":17670,"log count":9.7796235653},"Iraq":{"code":"iq","display":"Iraq","count":11961,"log count":9.389406636},"Ireland":{"code":"ie","display":"Ireland","count":277339,"log count":12.5329958636},"Isle of Man":{"code":"im","display":"Isle of Man","count":5658,"log count":8.6408257519},"Israel":{"code":"il","display":"Israel","count":191195,"log count":12.1610491286},"Italy":{"code":"it","display":"Italy","count":360760,"log count":12.7959681963},"Jamaica":{"code":"jm","display":"Jamaica","count":16679,"log count":9.7219057221},"Japan":{"code":"jp","display":"Japan","count":508835,"log count":13.139879078},"Jersey":{"code":"je","display":"Jersey","count":8780,"log count":9.0802316866},"Jordan":{"code":"jo","display":"Jordan","count":14446,"log count":9.5781728386},"Kazakhstan":{"code":"kz","display":"Kazakhstan","count":28251,"log count":10.2488841342},"Kenya":{"code":"ke","display":"Kenya","count":13568,"log count":9.515469358},"Kiribati":{"code":"ki","display":"Kiribati","count":1363,"log count":7.2174434317},"North Korea":{"code":"kp","display":"Democratic People's Republic of Korea","count":40461,"log count":10.6080938262},"South Korea":{"code":"kr","display":"Republic of Korea","count":935227,"log count":13.7485445596},"Kosovo":{"code":"xk","display":"Kosovo","count":7587,"log count":8.9341915353},"Kuwait":{"code":"kw","display":"Kuwait","count":14014,"log count":9.5478121089},"Kyrgyzstan":{"code":"kg","display":"Kyrgyzstan","count":6219,"log count":8.7353644011},"Laos":{"code":"la","display":"Laos","count":1980,"log count":7.5908521237},"Latvia":{"code":"lv","display":"Latvia","count":38259,"log count":10.5521341057},"Lebanon":{"code":"lb","display":"Lebanon","count":9702,"log count":9.1800873288},"Lesotho":{"code":"ls","display":"Lesotho","count":843,"log count":6.736966958},"Liberia":{"code":"lr","display":"Liberia","count":1242,"log count":7.1244782625},"Libya":{"code":"ly","display":"Libya","count":3551,"log count":8.1749845329},"Liechtenstein":{"code":"li","display":"Liechtenstein","count":1683,"log count":7.4283331942},"Lithuania":{"code":"lt","display":"Lithuania","count":35884,"log count":10.4880467927},"Luxembourg":{"code":"lu","display":"Luxembourg","count":8649,"log count":9.0651989863},"Macao":{"code":"mo","display":"Macao","count":9218,"log count":9.1289133733},"Madagascar":{"code":"mg","display":"Madagascar","count":5225,"log count":8.5612100768},"Malawi":{"code":"mw","display":"Malawi","count":1542,"log count":7.3408355541},"Malaysia":{"code":"my","display":"Malaysia","count":180472,"log count":12.1033309201},"Maldives":{"code":"mv","display":"Maldives","count":2335,"log count":7.7557671701},"Mali":{"code":"ml","display":"Mali","count":1885,"log count":7.5416830999},"Malta":{"code":"mt","display":"Malta","count":6847,"log count":8.8315658791},"Marshall Islands":{"code":"mh","display":"Marshall Islands","count":1323,"log count":7.1876571641},"Martinique":{"code":"mq","display":"Martinique","count":3260,"log count":8.0894824744},"Mauritania":{"code":"mr","display":"Mauritania","count":857,"log count":6.7534379186},"Mauritius":{"code":"mu","display":"Mauritius","count":4218,"log count":8.347116361},"Mayotte":{"code":"yt","display":"Mayotte","count":1081,"log count":6.9856418176},"Mexico":{"code":"mx","display":"Mexico","count":394958,"log count":12.8865347091},"Micronesia, Federated States of":{"code":"fm","display":"Micronesia, Federated States of","count":1234,"log count":7.1180162045},"Moldova":{"code":"md","display":"Moldova","count":9689,"log count":9.1787465004},"Monaco":{"code":"mc","display":"Monaco","count":2258,"log count":7.7222347447},"Mongolia":{"code":"mn","display":"Mongolia","count":7693,"log count":8.9480661035},"Montenegro":{"code":"me","display":"Montenegro","count":2831,"log count":7.9483852851},"Montserrat":{"code":"ms","display":"Montserrat","count":1206,"log count":7.0950643773},"Morocco":{"code":"ma","display":"Morocco","count":36350,"log count":10.500949483},"Mozambique":{"code":"mz","display":"Mozambique","count":2070,"log count":7.6353038863},"Myanmar":{"code":"mm","display":"Myanmar","count":4825,"log count":8.4815660138},"Namibia":{"code":"na","display":"Namibia","count":2540,"log count":7.83991936},"Nauru":{"code":"nr","display":"Nauru","count":739,"log count":6.6052979209},"Nepal":{"code":"np","display":"Nepal","count":8530,"log count":9.0513446405},"Netherlands":{"code":"nl","display":"Netherlands","count":425787,"log count":12.9616945002},"Netherlands Antilles":{"code":"an","display":"Netherlands Antilles","count":3624,"log count":8.1953336672},"New Caledonia":{"code":"nc","display":"New Caledonia","count":3106,"log count":8.0410910037},"New Zealand":{"code":"nz","display":"New Zealand","count":477570,"log count":13.076466025},"Nicaragua":{"code":"ni","display":"Nicaragua","count":9326,"log count":9.1405614774},"Niger":{"code":"ne","display":"Niger","count":9787,"log count":9.1888102534},"Nigeria":{"code":"ng","display":"Nigeria","count":30339,"log count":10.3201892926},"Niue":{"code":"nu","display":"Niue","count":1052,"log count":6.9584483933},"Norfolk Island":{"code":"nf","display":"Norfolk Island","count":1231,"log count":7.1155821262},"Macedonia, The Former Yugoslav Republic of":{"code":"mk","display":"North Macedonia","count":8257,"log count":9.0188166044},"Northern Mariana Islands":{"code":"mp","display":"Northern Mariana Islands","count":967,"log count":6.8741984955},"Norway":{"code":"no","display":"Norway","count":182347,"log count":12.1136667442},"Oman":{"code":"om","display":"Oman","count":11772,"log count":9.3734791094},"Pakistan":{"code":"pk","display":"Pakistan","count":67184,"log count":11.1151904029},"Palau":{"code":"pw","display":"Palau","count":1558,"log count":7.3511582264},"Palestine":{"code":"ps","display":"Palestine, State of","count":14711,"log count":9.5963507923},"Panama":{"code":"pa","display":"Panama","count":13240,"log count":9.4909978295},"Papua New Guinea":{"code":"pg","display":"Papua New Guinea","count":1719,"log count":7.4494980054},"Paraguay":{"code":"py","display":"Paraguay","count":6067,"log count":8.7106195279},"Peru":{"code":"pe","display":"Peru","count":90536,"log count":11.4135028406},"Philippines":{"code":"ph","display":"Philippines","count":93388,"log count":11.4445181363},"Pitcairn":{"code":"pn","display":"Pitcairn","count":1161,"log count":7.0570369817},"Poland":{"code":"pl","display":"Poland","count":1445666,"log count":14.184080673},"Portugal":{"code":"pt","display":"Portugal","count":217054,"log count":12.2879014495},"Puerto Rico":{"code":"pr","display":"Puerto Rico","count":18633,"log count":9.8326894812},"Qatar":{"code":"qa","display":"Qatar","count":19808,"log count":9.8938411755},"Reunion":{"code":"re","display":"Reunion","count":6752,"log count":8.8175940363},"Romania":{"code":"ro","display":"Romania","count":145494,"log count":11.8878901276},"Russia":{"code":"ru","display":"Russia","count":232246,"log count":12.3555524337},"Rwanda":{"code":"rw","display":"Rwanda","count":1833,"log count":7.5137092478},"Saint Barthelemy":{"code":"bl","display":"Saint Barth\u00e9lemy","count":688,"log count":6.5337888379},"Saint Helena":{"code":"sh","display":"Saint Helena, Ascension and Tristan Da Cunha","count":725,"log count":6.5861716549},"Saint Kitts and Nevis":{"code":"kn","display":"Saint Kitts and Nevis","count":976,"log count":6.8834625864},"Saint Lucia":{"code":"lc","display":"Saint Lucia","count":1142,"log count":7.0405363902},"Saint Martin":{"code":"mf","display":"Saint Martin","count":956,"log count":6.8627579131},"Saint Pierre and Miquelon":{"code":"pm","display":"Saint Pierre and Miquelon","count":499,"log count":6.2126060958},"Saint Vincent and the Grenadines":{"code":"vc","display":"Saint Vincent and the Grenadines","count":101,"log count":4.6151205168},"Samoa":{"code":"ws","display":"Samoa","count":4528,"log count":8.4180356199},"San Marino":{"code":"sm","display":"San Marino","count":1200,"log count":7.0900768358},"Sao Tome and Principe":{"code":"st","display":"Sao Tome and Principe","count":747,"log count":6.6160651851},"Saudi Arabia":{"code":"sa","display":"Saudi Arabia","count":40696,"log count":10.6138850865},"Senegal":{"code":"sn","display":"Senegal","count":3378,"log count":8.1250390974},"Serbia":{"code":"rs","display":"Serbia","count":48092,"log count":10.7808711221},"Seychelles":{"code":"sc","display":"Seychelles","count":1339,"log count":7.1996783457},"Sierra Leone":{"code":"sl","display":"Sierra Leone","count":1257,"log count":7.1364832086},"Singapore":{"code":"sg","display":"Singapore","count":109322,"log count":11.6020529348},"Sint Maarten":{"code":"sx","display":"Sint Maarten","count":72,"log count":4.276666119},"Slovakia":{"code":"sk","display":"Slovakia","count":46559,"log count":10.7484756045},"Slovenia":{"code":"si","display":"Slovenia","count":61105,"log count":11.0203489749},"Solomon Islands":{"code":"sb","display":"Solomon Islands","count":844,"log count":6.7381524946},"Somalia":{"code":"so","display":"Somalia","count":9438,"log count":9.1524993723},"South Africa":{"code":"za","display":"South Africa","count":78148,"log count":11.2663597437},"South Georgia and the South Sandwich Islands":{"code":"gs","display":"South Georgia and the South Sandwich Islands","count":4891,"log count":8.4951520605},"South Sudan":{"code":"ss","display":"South Sudan","count":1189,"log count":7.0808678967},"Spain":{"code":"es","display":"Spain","count":1312775,"log count":14.0876537753},"Sri Lanka":{"code":"lk","display":"Sri Lanka","count":14971,"log count":9.6138702754},"Sudan":{"code":"sd","display":"Sudan","count":3174,"log count":8.0627479011},"Suriname":{"code":"sr","display":"Suriname","count":1172,"log count":7.0664669701},"Svalbard and Jan Mayen":{"code":"sj","display":"Svalbard and Jan Mayen","count":900,"log count":6.8023947633},"Sweden":{"code":"se","display":"Sweden","count":433064,"log count":12.9786408021},"Switzerland":{"code":"ch","display":"Switzerland","count":136047,"log count":11.8207556933},"Syria":{"code":"sy","display":"Syria","count":12964,"log count":9.4699315643},"Taiwan":{"code":"tw","display":"Taiwan","count":545315,"log count":13.2091188883},"Tajikistan":{"code":"tj","display":"Tajikistan","count":1949,"log count":7.5750716995},"Tanzania":{"code":"tz","display":"Tanzania","count":3661,"log count":8.2054916131},"Thailand":{"code":"th","display":"Thailand","count":238868,"log count":12.3836663771},"Timor-Leste":{"code":"tl","display":"Timor-Leste","count":1208,"log count":7.0967213785},"Togo":{"code":"tg","display":"Togo","count":2686,"log count":7.8958083771},"Tokelau":{"code":"tk","display":"Tokelau","count":1012,"log count":6.9196838498},"Tonga":{"code":"to","display":"Tonga","count":3428,"log count":8.1397322797},"Trinidad and Tobago":{"code":"tt","display":"Trinidad and Tobago","count":9209,"log count":9.1279365457},"Tunisia":{"code":"tn","display":"Tunisia","count":21832,"log count":9.9911320624},"Turkey":{"code":"tr","display":"Turkey","count":705946,"log count":13.4672940263},"Turkmenistan":{"code":"tm","display":"Turkmenistan","count":1611,"log count":7.3846103832},"Turks and Caicos Islands":{"code":"tc","display":"Turks and Caicos Islands","count":2264,"log count":7.7248884393},"Tuvalu":{"code":"tv","display":"Tuvalu","count":2655,"log count":7.8841999337},"Uganda":{"code":"ug","display":"Uganda","count":13512,"log count":9.5113334585},"Ukraine":{"code":"ua","display":"Ukraine","count":126112,"log count":11.74492568},"United Arab Emirates":{"code":"ae","display":"United Arab Emirates","count":123829,"log count":11.7266568606},"United Kingdom":{"code":"gb","display":"United Kingdom","count":2871231,"log count":14.8702514156},"United States":{"code":"us","display":"United States of America","count":19924957,"log count":16.8074836245},"United States Minor Outlying Islands":{"code":"um","display":"United States Minor Outlying Islands","count":182454,"log count":12.1142533654},"Uruguay":{"code":"uy","display":"Uruguay","count":61137,"log count":11.0208725265},"Uzbekistan":{"code":"uz","display":"Uzbekistan","count":6205,"log count":8.7331106976},"Vanuatu":{"code":"vu","display":"Vanuatu","count":2290,"log count":7.7363070965},"Vatican City":{"code":"va","display":"Vatican City","count":2767,"log count":7.9255189798},"Venezuela":{"code":"ve","display":"Venezuela","count":27878,"log count":10.2355931263},"Vietnam":{"code":"vn","display":"Vietnam","count":105419,"log count":11.5656981645},"Virgin Islands, British":{"code":"vg","display":"Virgin Islands, British","count":5573,"log count":8.6256887876},"Virgin Islands, U.S.":{"code":"vi","display":"Virgin Islands, U.S.","count":11751,"log count":9.3716936223},"Wallis and Futuna":{"code":"wf","display":"Wallis and Futuna","count":2209,"log count":7.7002952034},"Western Sahara":{"code":"eh","display":"Western Sahara","count":2255,"log count":7.7209052519},"Yemen":{"code":"ye","display":"Yemen","count":8123,"log count":9.0024548231},"Zambia":{"code":"zm","display":"Zambia","count":8026,"log count":8.9904415508},"Zimbabwe":{"code":"zw","display":"Zimbabwe","count":39471,"log count":10.583321504}}

```

File: src/components/people-grid/people-grid.jsx [hunk 7]
```diff
@@ -0,0 +1,46 @@
+const PropTypes = require('prop-types');
```
Comment (seotts) [inline src/components/people-grid/people-grid.jsx:1]: All of the people-grid files are in a separate PR that Ben is reviewing, so I don't think we need to look to closely at them here

File: src/views/credits/credits.jsx [hunk 8]
```diff
@@ -2,9 +2,9 @@ const React = require('react');
 const render = require('../../lib/render.jsx');
```
Comment (seotts) [inline src/views/credits/credits.jsx:2]: This is also in the separate people grid pr

**Review by rschamp — APPROVED** [review]
I just realized I never submitted these responses. Just submitting it now so it's not just pending. Nothing to act on here of course.

**Pull request merged**